hoot-core/src/main/cpp/hoot/core/io/OsmApiDbReader.h

```diff
@@ -45,7 +45,6 @@ class OsmApiDbReader :
 public:
 
   static std::string className() { return "hoot::OsmApiDbReader"; }
-  static QString emailKey() { return "osmapi.db.reader.email"; }
```

Comment:
Looks like here you just delete that function because it is no longer needed.  I'm thinking that is what we should do in the other two cases above.